.github/workflows/deploy.yml

```diff
@@ -50,14 +52,13 @@ jobs:
         run: |
           echo "PATH=$(pwd)/node_modules/.bin/:$PATH" >> $GITHUB_ENV
       - name: run unit tests
-        run: ./test-unit.sh {{ env.STAGE_PREFIX }}{{ env.branch_name }}
+        run: ./test-unit.sh
```

Comment:
Part of the changes needed to close #263 
Our unit testing framework was sitting on top of serverless, so a stage needed to be passed to the test runner.  Since we have ditched serverless as a unit test wrapper, passing the stage name (as seen as removed here) is no longer needed.